.nuget/FStar.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<package>
+	<metadata>
```

Comment:
@tahina-pro @nikswamy What should I put as metadata for the nuget? 

The tags included below are mandatory for a package to be uploaded to nuget.org. There are others though, like projectUrl, or license (some described here: https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package).

Reply:
I can be a backup owner, since I am at least responsible for the repository-agnostic binary package overall already

Reply:
From https://docs.microsoft.com/en-us/nuget/reference/nuspec, we can drop `<owners>`, which is deprecated in favor of `<authors>`. This field lists package maintainers, as NuGet account IDs. Mine is `tahina`